test/smoke/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -95,6 +99,45 @@ export function prepareReactNativeApplication(workspaceFilePath: string, resourc
     fs.writeFileSync(path.join(vsCodeConfigPath, "launch.json"), fs.readFileSync(launchConfigFile));
 }
 
+export function prepareExpoApplication(workspaceFilePath: string, resourcesPath: string, workspacePath: string, appName: string) {
+    const command = `echo -ne '\\n' | expo init -t tabs --name ${appName}  --workflow managed ${appName}`;
+    console.log(`*** Creating Expo app via '${command}' in ${workspacePath}...`);
+    cp.execSync(command, { cwd: resourcesPath, stdio: "inherit" });
+
+    let customEntryPointFile = path.join(resourcesPath, "ExpoSample", "App.js");
+    let launchConfigFile = path.join(resourcesPath, "launch.json");
+    let vsCodeConfigPath = path.join(workspacePath, ".vscode");
+
+    console.log(`*** Copying  ${customEntryPointFile} into ${workspaceFilePath}...`);
+    fs.writeFileSync(workspaceFilePath, fs.readFileSync(customEntryPointFile));
+
+    if (!fs.existsSync(vsCodeConfigPath)) {
+        console.log(`*** Creating  ${vsCodeConfigPath}...`);
+        fs.mkdirSync(vsCodeConfigPath);
+    }
+
+    console.log(`*** Copying  ${launchConfigFile} into ${vsCodeConfigPath}...`);
+    fs.writeFileSync(path.join(vsCodeConfigPath, "launch.json"), fs.readFileSync(launchConfigFile));
+}
+
+// Installs Expo app on Android device via Expo start command
+export async function installExpoAppOnAndroid(expoAppPath: string) {
+    console.log(`*** Installing Expo app (${expoPackageName}) on android device with 'expo-cli android' command`);
+    let expoCliCommand = process.platform === "win32" ? "expo-cli.cmd" : "expo-cli";
```

Comment:
Do we really need to use `expo android` command? Can't we use `expo install:android`?

Reply:
Oh, I'll fix this